product-cli/cmd/cluster/cluster.go

```diff
@@ -70,8 +70,6 @@ func NewCreateCommands() *cobra.Command {
 	cmd.PersistentFlags().BoolVar(&opts.Wait, "wait", opts.Wait, "If true, the create command will block until the HostedCluster is up. Requires at least one NodePool with at least one node.")
 	cmd.PersistentFlags().StringVar(&opts.PausedUntil, "pausedUntil", opts.PausedUntil, "If a date is provided in RFC3339 format, HostedCluster creation is paused until that date. If the boolean true is provided, HostedCluster creation is paused until the field is removed.")
 
-	_ = cmd.MarkPersistentFlagRequired("pull-secret")
```

Comment:
It the PS is not provided, MCE will use the loaded in the MGMT cluster?

Reply:
No, the pull secret needs to be provided either via `--pull-secret` or `--secret-creds`